lbrynet/reflector/server/server.py

```diff
@@ -107,6 +108,7 @@ def _on_completed_blob(self, blob, response_key):
             yield save_sd_info(self.stream_info_manager, sd_info)
             yield self.stream_info_manager.save_sd_blob_hash_to_stream(sd_info['stream_hash'],
                                                                        blob.blob_hash)
+            yield self.lbry_file_manager.add_lbry_file(sd_info['stream_hash'])
```

Comment:
This should be non-blocking (otherwise it would block sending the `blob_completed` response on the file being started)